image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -49,6 +49,27 @@ spec:
       {{- if ._rox.scanner.priorityClassName }}
       priorityClassName: {{ ._rox.scanner.priorityClassName }}
       {{- end }}
+      {{- if not ._rox.env.centralServices }}
+      initContainers:
+        - name: init-tls-certs
+          image: {{ quote ._rox.image.main.fullRef }}
+          command:
+          - /stackrox/bin/init-tls-certs
+          resources:
+            {{- ._rox.scanner._resources | nindent 12 }}
```

Comment:
Shouldn't this be filled in with something specific to the `init-tls-certs` use-case?
(Here and for the other services as well)

Reply:
Seems to be the convention used so far in the charts (initContainers use the same resources as the "main" container).

Reply:
Hmm. But does it make sense?

Reply:
This `init-tls-certs` initContainer most likely needs less resources than any of the containers that it initializes. 

But does adjusting its resources help? In the current situation we over provision. And if this leads to the initContainer not being scheduled, we just end up failing earlier (the "main" container would still not be scheduled). And we also have less hardcoded resources that we need to maintain.

On the other hand, I suppose that k8s would deal with the cluster resources better if we set more accurate resources for this container.

Reply:
BTW this is something that was in WIP PR of @porridge and I don't know if it was on purpose, or a quick hack that he intended to change later on.

Reply:
It was definitely a quick hack. However I don't think replacing this with a stanza that reserves less resources will change anything. IIRC pods are scheduled according to `max(init_containers, sum(non_init_containers))` so all that we would gain is a bit more available space on the node _while the init container is running_. Probably not worth bothering. 🤷🏻

Reply:
👍